templates/definition/vehicle/bmw.yaml

```diff
@@ -4,9 +4,9 @@ products:
 requirements:
   description:
     de: |
-      Benötigt `hcaptcha` Token. Dieses muss einmalig unter https://bimmer-connected.readthedocs.io/en/latest/captcha/rest&#95;of&#95;world.html generiert werden. Das Token ist nur für kurze Zeit gültig. Bitte möglichst schnell nach Generierung in die Konfiguration kopieren und evcc starten.
+      Benötigt `hcaptcha` Token. Dieses muss einmalig unter [https://bimmer-connected.readthedocs.io/en/latest/captcha/rest&#95;of&#95;world.html](https://bimmer-connected.readthedocs.io/en/latest/captcha/rest_of_world.html) generiert werden. Das Token ist nur für kurze Zeit gültig. Bitte möglichst schnell nach Generierung in die Konfiguration kopieren und evcc starten.
```

Comment:
Ist das ein gültiger, anklickbarer Link? Semikolons in Urls sind egtl. nicht zulässig.

Reply:
Ja - die eckigen Klammern enthalten den Text, die runden Klammern enthalten den Link.

HTML-Code im Browser:
```html
<a href="https://bimmer-connected.readthedocs.io/en/latest/captcha/rest_of_world.html">https://bimmer-connected.readthedocs.io/en/latest/captcha/rest_of_world.html</a>
```